core/src/test/java/tech/tablesaw/api/FixDropDuplicatesTest.java

```diff
@@ -0,0 +1,38 @@
+package tech.tablesaw.api;
+
+import java.io.File;
+import java.lang.reflect.Method;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import it.unimi.dsi.fastutil.ints.IntArrayList;
+import tech.tablesaw.io.csv.CsvReadOptions;
+
+class FixDropDuplicatesTest {
+
+    private static final String SOURCE_FILE_NAME = "../data/missing_values.csv";
+    private static Table testTable;
+    
+    @BeforeAll
+    static void setUpBeforeClass() throws Exception {
+        testTable = Table.read().usingOptions(CsvReadOptions
+            .builder(new File(SOURCE_FILE_NAME))
+            .missingValueIndicator("-"));    }
+
+    @Test
+    void test() throws Exception {
+        Method privateMethod = Table.class.getDeclaredMethod("isDuplicate", Row.class, Int2ObjectMap.class);
```

Comment:
I'd prefer to just make it package visible for testing

Or perhaps better would be to test `dropDuplicateRows` like the other duplicate tests are doing if possible. I haven't closely checked the code itself to understand the issue yet. Is it really hard to reproduce the issue from the public APIs? (a  bit of description of the issue in the PR might help as well since it has been awhile since I've looked at this codebase)

Reply:
The issue only happens when non duplicate row hashes collide, it's pretty hard to test even with mocking.

As you prefer for method visibility, I personally prefer to keep things private.

Reply:
The issue description is detailed in #1248, do you prefer I copy it there ?

Reply:
Ah, sorry, I hadn't taken a look at the issue. No need to copy that into the PR

Reply:
There are a couple methods we've annotated with `@VisibleForTesting`. You could do that here when changing the visibility